src/Controller/BannerController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use WMDE\BannerServer\UseCase\BannerProvideRequest;
+use WMDE\BannerServer\UseCase\BannerProvideUseCase;
+
+/**
+ * @license GNU GPL v2+
+ */
+class BannerController {
+
+	public function provide(): Response {
+		$useCase = new BannerProvideUseCase();
```

Comment:
The `new` here will keep you from being able to inject a mocked use case to check if the parameters were passed in right. Have a look at https://symfony.com/doc/current/controller.html#controller-accessing-services, register the use case as a service and inject it via parameter to `provide`.